packages/ui/src/machines/authenticator/actions.ts

```diff
@@ -66,7 +66,8 @@ export const setConfirmSignUpIntent = assign({
 export const setCredentials = assign({
   authAttributes: (context: SignInContext | SignUpContext, _) => {
     const [primaryAlias] = context.loginMechanisms;
-    const username = context.formValues[primaryAlias];
+    const username =
+      context.formValues[primaryAlias] ?? context.formValues['username'];
```

Comment:
The actual fix is here. Ideally, we would enforce better input `name` conventions but that's a bigger refactor.